Sources/Sentry/SentryProfiler.mm

```diff
@@ -469,15 +469,19 @@ + (nullable SentryEnvelopeItem *)createEnvelopeItemForProfilePayload:
     }
 
     if ([fm fileExistsAtPath:pathToWrite]) {
-        SENTRY_LOG_DEBUG(@"Already a%@ profile file present; make sure to remove them right after "
+        SENTRY_LOG_DEBUG(@"Already a %@ profile file present; make sure to remove them right after "
                          @"using them, and that tests clean state in between so there isn't "
                          @"leftover config producing one when it isn't expected.",
             isTracingAppLaunch ? @" launch" : @"");
+        return;
```

Comment:
Dang good catch, I meant to add an early return here. The extra space before the %@ is unnecessary though, I know it's a bit weird.